TypeScript: Fine-Grained Assertion Tests HitCounter 'DynamoDB Table Created' test missing lambda import

## :bug: Bug Report

### What is the problem?
When following the instructions on ASSERTION TESTS for the TypeScript workshop the first time you run ```npm run build && npx jest``` it fails, reporting Cannot find name 'lambda'.

### Reproduction Steps
Copy hitcounter.test.ts code from workshop
Run npm run build && npx jest

### Verbose Log
 cdk-workshop git:(master) ✗ npm run build && npx jest

> cdk-workshop@0.1.0 build /cdk-workshop
> tsc

test/hitcounter.test.ts:9:26 - error TS2304: Cannot find name 'lambda'.

9         downstream:  new lambda.Function(stack, 'TestFunction', {
                           ~~~~~~
npm ERR! code ELIFECYCLE
npm ERR! errno 2
npm ERR! cdk-workshop@0.1.0 build: `tsc`
npm ERR! Exit status 2
npm ERR! 
npm ERR! Failed at the cdk-workshop@0.1.0 build script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:


### Environment

  - **CDK CLI Version:** 1.51.0 (build 8c2d53c)
  - **Section:** Fine-Grained Assertion Tests
  - **Browser:** N/A
  - **Language:** TypeScript

### Other information 
Including
```import * as lambda from '@aws-cdk/aws-lambda'; ``` during the imports appears to solve the problem

--- a/workshop/content/20-typescript/70-advanced-topics/100-construct-testing/1000-assertion-test.md
+++ b/workshop/content/20-typescript/70-advanced-topics/100-construct-testing/1000-assertion-test.md
@@ -19,6 +19,7 @@ same level as `bin` and `lib` and then create a file called `hitcounter.test.ts`
 import { expect as expectCDK, haveResource } from '@aws-cdk/assert';
 import cdk = require('@aws-cdk/core');
 import * as lambda from '@aws-cdk/aws-lambda';
+
 import { HitCounter }  from '../lib/hitcounter';
 
 test('DynamoDB Table Created', () => {

--- a/workshop/content/20-typescript/70-advanced-topics/100-construct-testing/1000-assertion-test.md
+++ b/workshop/content/20-typescript/70-advanced-topics/100-construct-testing/1000-assertion-test.md
@@ -18,6 +18,7 @@ same level as `bin` and `lib` and then create a file called `hitcounter.test.ts`
 ```typescript
 import { expect as expectCDK, haveResource } from '@aws-cdk/assert';
 import cdk = require('@aws-cdk/core');
+import * as lambda from '@aws-cdk/aws-lambda';
 import { HitCounter }  from '../lib/hitcounter';
 
 test('DynamoDB Table Created', () => {

